### ISSUE TITLE
Write set up script for toolchain

### ISSUE THREAD

**Issue by codeDavidLopez** [amosproj/amos2022ss06-idunion-blockchain-dashboard#59]
## User story
1. As a developer
2. I want / need a set up script
3. So that the toolchain is set up reproducable

## Acceptance criteria
* script for set up available (e. g. shell script, docker file)
* every developer can set up with the script the toolchain (comment the script)
* toolchain set up reproducable without any individual changes or changes with parameter

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## Feature DoD 
* Feature has been fully implemented
* All acceptance criteria were met
* All tests are passing
* No known errors
* Documentation updated
* Bill of materials updated

## Sprint Release DoD
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Peer code review passed
* Developers agreed to release
* Documentation updated
* Bill of materials updated
